src/util/pointInImage.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * PointInImage - Checks if the point is within the image boundaries.
+ * @param  {Object} pixel The pixel.
+ * @param  {number} rows The number of rows.
+ * @param  {number} cols The number of columns.
+ */
+export default function(pixel, rows, cols) {
```

Comment:
`isPointInImage`

- destructure `pixel` in constructor?
- export a named function, and remove function name from desc
    - Better JSDoc, still searchable with `ctrl` + `shift` + `f`